### PR TITLE
Allow multiple CORS origins to be specified

### PR DESCRIPTION
**Pull request by burleight**
We'd find it useful if we could specify multiple, comma separated origins when using the `--cors-origin` flag

### CODE REVIEW

File: src/test/java/de/komoot/photon/ApiIntegrationTest.java [hunk 1]
```diff
@@ -72,6 +73,57 @@ void testCorsOriginIsSetToSpecificDomain() throws Exception {
...
+        try {
+            // allows us to set the Origin header
+            System.setProperty("sun.net.http.allowRestrictedHeaders", "true");
+
+            String[] origins = {"www.randomsite.com", "www.arbitrary.com"};
+            for (String origin: origins) {
+                URLConnection urlConnection = new URL("http://127.0.0.1:" + port() + "/api?q=berlin").openConnection();
+                urlConnection.setRequestProperty("Origin", origin);
+                HttpURLConnection connection = (HttpURLConnection) urlConnection;
+                assertNotEquals(origin, connection.getHeaderField("Access-Control-Allow-Origin"));
```
Comment (lonvia) [inline src/test/java/de/komoot/photon/ApiIntegrationTest.java]: If I read the code right, this should return the first configured origin. So better test for `assertEquals("www.poole.ch", ...`.

**Review by lonvia — COMMENTED** [review]
Looks good to me in principle. Clean code and the tests are all there.

Not sure why the tests don't pass.

File: src/test/java/de/komoot/photon/ApiIntegrationTest.java (hunk 1 above)
Comment (burleight) [reply src/test/java/de/komoot/photon/ApiIntegrationTest.java]: Thanks for the feedback.

I've made this change, and fixed the tests with

```
test {
    systemProperty "sun.net.http.allowRestrictedHeaders", "true"
}
```

**Comment by burleight** [comment]
@lonvia  - hopefully this third try does it. I've moved the test configuration into `shared.gradle`

**Comment by lonvia** [comment]
It did indeed. Thank you!

**Pull request merged**